MapboxDirections/MBVisualInstructionType.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+/**
+ `VisualInstructionComponentType` describes the type of `VisualInstructionComponent`.
+ */
+@objc(MBVisualInstructionComponentType)
+public enum VisualInstructionComponentType: Int, CustomStringConvertible {
+    
+    /**
+     Text is a delimiter.
+     */
+    case delimiter
+    
+    /**
+     Text is a way name.
```

Comment:
A destination isn’t necessarily the name of a street:

> The component bears an exit number or the name of a place or street.

Come to think of it, it would be nice to be able to distinguish exit numbers from place or street names. 🤔

Reply:
I agree, but more changes will need to be made on the server.

Reply:
In the meantime, I think this description is accurate:

> The component bears an exit number or the name of a place or street.